index.d.ts

```diff
@@ -13,6 +13,7 @@ import {
 export function FontAwesomeIcon(props: Props): JSX.Element
 
 export interface Props {
+  ref: ((e: any) => void) | React.RefObject<any>
```

Comment:
```suggestion
  ref?: ((e: any) => void) | React.RefObject<any>
```

`ref` should not be a required property.